srcpkgs/python3-ruamel.yaml/template

```diff
@@ -1,25 +1,23 @@
 # Template file for 'python3-ruamel.yaml'
 pkgname=python3-ruamel.yaml
-version=0.16.12
-revision=4
+version=0.17.21
+revision=1
 wrksrc="ruamel.yaml-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
-depends="python3"
+depends="python3-ruamel.yaml.clib"
 short_desc="YAML parser/emitter in Python 3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="http://yaml.readthedocs.io/"
+changelog="https://sourceforge.net/p/ruamel-yaml/code/ci/default/tree/CHANGES"
 distfiles="${PYPI_SITE}/r/ruamel.yaml/ruamel.yaml-${version}.tar.gz"
-checksum=076cc0bc34f1966d920a49f18b52b6ad559fbe656a0748e3535cf7b3f29ebf9e
+checksum=8b7ce697a2f212752a35c1ac414471dc16c424c9573be4926b56ff3f5d23b7af
+make_check=no # no tests included in distfiles
```

Comment:
could we use a distfiles source that has tests?

Reply:
Not sure if it's possible. It's hosted on sourceforge.net but there are no tarballs/downloads in https://sourceforge.net/projects/ruamel-yaml/files/ like there are for other packages with distfiles from sourceforge, e.g. https://sourceforge.net/projects/drumkv1/files/